src/Java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/GregtechMetaTileEntity_IsaMill.java

```diff
@@ -344,7 +319,7 @@ public Block getIntakeBlock() {
 	}
 
 	public byte getIntakeMeta() {
-		return 0;
+		return 1;
```

Comment:
it is 0 here in old code. the isamill pipe casing even can't change its texture when it is around the controller.

Reply:
And? They're visible from the side and there's a good chance I just never finished the structure check.. why else would I have added them?